lib/categories_screen.dart

```diff
@@ -0,0 +1,34 @@
+import 'package:flutter/material.dart';
+import 'package:meals_app/category_item.dart';
+import 'package:meals_app/dummy_data.dart';
+
+class CategoriesScreen extends StatelessWidget {
+  const CategoriesScreen({super.key});
+
+  @override
+  Widget build(BuildContext context) {
+    final theme = Theme.of(context);
+    return Scaffold(
+      appBar: AppBar(
+        elevation: 5,
+          title: const Text('DeliMeal'),
+          backgroundColor: theme.primaryColor,
+      ),
+      body: Container(
+        color: theme.canvasColor,
+        child: GridView(
+          padding: const EdgeInsets.all(15),
+          gridDelegate: const SliverGridDelegateWithMaxCrossAxisExtent(
+              maxCrossAxisExtent: 200,
+              childAspectRatio: 3 / 2,
+              crossAxisSpacing: 20,
+              mainAxisSpacing: 20),
```

Comment:
Just a number in code is called a Magic number. Usually I prefer to create a constant for this so that we can give it a clear name and we can reuse it in the future